doc/users/event_handling.rst

```diff
@@ -194,43 +180,41 @@ Here is the solution::
                 'motion_notify_event', self.on_motion)
 
         def on_press(self, event):
-            'on button press we will see if the mouse is over us and store some data'
-            if event.inaxes != self.rect.axes: return
-
+            """Check whether mouse is over us; if so, store some data."""
+            if event.inaxes != self.rect.axes:
+                return
             contains, attrd = self.rect.contains(event)
-            if not contains: return
+            if not contains:
+                return
             print('event contains', self.rect.xy)
-            x0, y0 = self.rect.xy
-            self.press = x0, y0, event.xdata, event.ydata
+            self.press = self.rect.xy, event.xdata, event.ydata
```

Comment:
This would be `(x0, y0), event.xdata, event.ydata`. Is this change intentional?

Edit: Seems like. But then I'd go for `self.rect.xy, (event.xdata, event.ydata)` so that below you have `(x0, y0), (xpress, ypress) = self.press`. Using a tuple only for one of the coordinates would be a bit awkward.